CHANGELOG.md

```diff
@@ -4,6 +4,14 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
+## 1.2.1 (unreleased)
+
+**Notable commits**
+
+**Thanks**
+
+* James Sewell for helping debug a segfault in last()
```

Comment:
Do we have / is it possible to ask for their github name?  That's at least been the practice so far so their account gets cross-referenced in the release notes.